### DISCUSSION TITLE
Lost all the project files on my local folder in my pc.. HELP!

### DISCUSSION THREAD

**Discussion by asheshgoplani** [community/community#21854] [New to GitHub]
<p>Hello, <br>
I’m new on github, so i had a project that i started working on in the start of november so i worked on it and then commited the changes on github after that i didnt use github at all didnt commit anything and continued working on the project so at the end of the november my project was completed but i still didnt commit any changes to github, after that i started working on some other project<br>
Now, today when i opened that orignal project that was saved and working at the end of november, what i found that it was overwritten by at the start of december, with a previous version that was in the start of november so all the changes i’ve done in the project are now lost.<br>
Kindly, if anyone has any idea how could i get back my project, please feel free to share</p>
<p>Thanks a lot</p>

**Comment by Wabri** [comment]
<p>Hi <a class="mention" href="/asheshgoplani">@asheshgoplani</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>This is a real problem, if you don’t push to github your project remain into your local machine and if you override the project whit an old version, I’m sorry for you, but is probably lost.</p>
<p>I see that you have more projects on github but in every project you only have 1 commit, do you know how to push?</p>
<p>Ask if you have questions to do :slight_smile:</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>

**Comment by asheshgoplani** [comment, marked as answer]
<p>Hello, <br>
Got all the files back, actually they were all stored in stash <br>
Thanks</p>
